CHANGELOG.md

```diff
@@ -3,7 +3,9 @@
 
 ## v1.10.0
 
-- Add metadata to TopicPartition type and commit() (#1410).
+ - Added metadata to TopicPartition type and commit() (#1410).
```

Comment:
hint: use backticks to make code stand out more, it looks nicer and makes it easier for the reader to identify what is code, e.g.:

Added metadata to `TopicPartition` type and `commit()`

..

Added `conusmer.memberid()`..

Reply:
Done